jskit/encoders/bi.py

```diff
@@ -101,16 +101,48 @@ def cosine_sim(vec_a: List[float], vec_b: List[float], meta):
 
 
 @jaseci_action(act_group=['bi_enc'], allow_remote=True)
-def infer(contexts: List, candidates: List):
+def infer(contexts: Union[List[str], List[List]], candidates: Union[List[str], List[List]], context_type: str, candidate_type: str):
     """
     Take list of context, candidate and return nearest candidate to the context
     """
     global model
     model.eval()
-    predicted_candidate = get_inference(model, tokenizer,
-                                        contexts=contexts,
-                                        candidates=candidates)
-    return predicted_candidate
+    predicted_candidates = []
+    if (context_type == "text") and (candidate_type == "text"):
+        predicted_candidates = get_inference(model, tokenizer,
+                                             contexts=contexts,
+                                             candidates=candidates)
+    elif (context_type == "text") and (candidate_type == "embedding"):
+        con_embed = []
```

Comment:
`con_embed` is being overwritten immediately on the next line so this line is not necessary right?